docs/usage/quickstart.rst

```diff
@@ -5,33 +5,93 @@
    Todo
    usernames
 
+
 Quickstart Guide
 ================
-To quickly set up a testing cluster using MetalK8s_, you need 3 machines
-running CentOS_ 7.4 to which you have SSH access (these can be VMs). Each
-machine acting as a Kubernetes_ node (all of them, in this example) also
-need to have at least one disk available to provision storage volumes.
-
-.. todo:: Give some sizing examples
+This guide describes how to set up a MetalK8s_ cluster. It offers general
+requirements and describes sizing, configuration, and deployment. With respect
+to installation procedures, the only significant difference between a test
+cluster and a full production environment is the amount of resources required
+to implement the design.
 
 .. _MetalK8s: https://github.com/scality/metal-k8s/
 .. _CentOS: https://www.centos.org
 .. _Kubernetes: https://kubernetes.io
 
-Defining an Inventory
----------------------
-To tell the Ansible_-based deployment system on which machines MetalK8s should
-be installed, a so-called *inventory* needs to be provided. This inventory
-contains a file listing all the hosts comprising the cluster, as well as some
-configuration.
+General Cluster Requirements
+----------------------------
+Setting up a MetalK8s cluster quickly requires at least three machines
+running CentOS_ 7.4 or higher (these can be VMs) to which you have SSH access.
+Each machine acting as a Kubernetes_ node (all three in the present example)
+must also have at least one disk available to provision storage volumes.
+
+Sizing
+^^^^^^
+
+Each node must satisfy the following sizing requirements.
+
+.. note::
+   The root file system requires at least 20 GB.
+
++-----------------+--------+--------+-------+
+|    Component    | etcd   | Master | Node  |
++=================+========+========+=======+
+| Cores           | 2      | 4      | 4     |
++-----------------+--------+--------+-------+
+| RAM             | 4 GB   | 8 GB   | 8 GB  |
++-----------------+--------+--------+-------+
+| Minimum         |        |        |       |
+| dedicated       |        |        |       |
+| storage capacity|        |        |       |
+| required        |    -   |    -   | 128 GB|
++-----------------+--------+--------+-------+
+
+Proxies
+^^^^^^^
+For nodes operating behind a proxy, add the following lines to each cluster
+server's :file:`/etc/environment` file:
+
+.. code-block:: shell
+
+ http_proxy=http://user;pass@<HTTP proxy IP address>:<port>
+ https_proxy=http://user;pass@<HTTPS proxy IP address>:<port>
+ no_proxy=localhost,127.0.0.1,10.*
```

Comment:
This is likely incorrect, but OK for now until we are sure how to do it 'properly'.

Also, we may want to let MetalK8s handle this in 1.1.